.github/workflows/rust.yml

```diff
@@ -78,6 +78,7 @@ jobs:
       - name: benchmark (smoke)
         run: |
           cargo run --package sui-benchmark --bin stress -- --log-path /tmp/stress.log --num-client-threads 10 --num-server-threads 24 --num-transfer-accounts 2 bench --target-qps 100 --num-workers 10  --transfer-object 50 --shared-counter 50 --run-duration 10s
+          pushd narwhal/benchmark && fab local && popd
```

Comment:
Should this be `fab smoke` or something that uses the new mode?

Reply:
Thanks for catching this! Should be `smoke` 🤦🏽